### ISSUE TITLE
Weight file for custom training

### ISSUE THREAD

**Issue by liwanjie1020** [eriklindernoren/PyTorch-YOLOv3#184]
Hello, I am a novice, I follow your steps to train my own data set, but after the training, I do not know where the weight file generated, I have no way to test, I hope you can help me solve it

It may be a simple question, but I need your help. thank you
![result](https://user-images.githubusercontent.com/41810006/58143179-74dde980-7c7c-11e9-94b1-8458dfabb031.jpg)

**Comment by tc-qaq** [comment]
At root/checkpoints/

**Comment by liwanjie1020** [comment]
oh，thank you very much

------------------
北京理工大学2018级软件工程专业研究生李婉洁







------------------ 原始邮件 ------------------
发件人: "汤成"<notifications@github.com>;
发送时间: 2019年5月27日(星期一) 中午11:19
收件人: "eriklindernoren/PyTorch-YOLOv3"<PyTorch-YOLOv3@noreply.github.com>;
抄送: "Fighting……"<1046836052@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [eriklindernoren/PyTorch-YOLOv3] Weight file for custom training(#184)




At root/checkpoints/

**Comment by yyx1107** [comment]
Hi~How did you to set the hyperparameters, such as batchsize.
I use the VOC2007 datasets ,but it's total loss  couldn't converge very well and the total loss is unstable.(between about 1.5~2). Hope for your reply(*￣︶￣) (Or  can you add me 1017176539)Thank you very much~
@liwanjie1020
